File: Estrutura_Condicional/Python/Ex2313.py
a, b, c = map(int, input().split())

if a >= b + c or b >= a + c or c >= a + b:
    print("Invalido")
else:
    if a**2 == b**2 + c**2 or b**2 == a**2 + c**2 or c**2 == a**2 + b**2:
        retangulo = "S"
    else:
        retangulo = "N"
    
    if a == b == c:
        print("Valido-Equilatero")
    elif a == b or a == c or b == c:
        print("Valido-Isoceles")
    else:
        print("Valido-Escaleno")
    
    print("Retangulo:", retangulo)

File: Estrutura_Condicional/JavaScript/Ex2313.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var prompt = function(texto) { return lines.shift();};

var entrada = prompt().split(" ");
var a = parseInt(entrada[0]);
var b = parseInt(entrada[1]);
var c = parseInt(entrada[2]);

if (a >= b + c || b >= a + c || c >= a + b) {
    console.log("Invalido");
} else {
    var retangulo;
    if (a**2 === b**2 + c**2 || b**2 === a**2 + c**2 || c**2 === a**2 + b**2) {
        retangulo = "S";
    } else {
        retangulo = "N";
    }
    
    if (a === b && b === c) {
        console.log("Valido-Equilatero");
    } else if (a === b || a === c || b === c) {
        console.log("Valido-Isoceles");
    } else {
        console.log("Valido-Escaleno");
    }
    
    console.log("Retangulo:", retangulo);
}